string ades_name
---
KeyValPair[] effects
